identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -37,15 +37,29 @@ sign.up.heading=Sign Up
 sign.up.button=Sign Up
 # <!-- End of Sign Up Screen Translations -->
 
+# <!-- SMS OTP -->
+sms.otp.heading=OTP Verification
+enter.code.sent.smsotp=Enter the code sent to your mobile phone
+sms.otp.resend.code=Resend Code
+sms.otp.submit.button=Continue
+
 # <!-- Start of Password Recovery Translations -->
 # EDITABLE=true,SCREEN="password-recovery",MULTI_LINE=false
 password.recovery.heading=Forgot Password?
 # EDITABLE=true,SCREEN="password-recovery",MULTI_LINE=true
-password.recovery.body=Don't worry, it happens. We will send you an email to reset your password.
+password.recovery.body=No problem, we're here to assist. Let's get your password reset for you.
```

Comment:
validate the content with the doc team. For me this content seems like bit too casual

Reply:
IMO, the original message sounds reassuring which is a positive in this context. I am +1 for that too.

But, If we are to be slightly formal we can go with,
"We've got you covered. Click the button below to receive a reset link to your email."

alternatively, remove "We've got you covered" to make it formal.
"Click the button below to receive a reset link to your email".

Reply:
This text is common for both email-link and sms-otp options. Hence we will have to provide a generic text here.

Reply:
How about "We've got you covered. Please submit below form to begin resetting your password."

Reply:
"We've got you covered. Enter your username to begin resetting your password."

Reply:
looks good to me. @Achintha444 what do you. think?

Reply:
fixed with https://github.com/wso2/identity-apps/pull/5817/commits/e4cdf46730b8f42c8b52051dc81c9a338c3f9110